node/src/main/resources/defaults.conf

```diff
@@ -291,6 +291,8 @@ casper {
     # Public key for transfers from the PoS vault
     # This public key is used to make transfer from PoS vault defined in PoS contract
     pos-vault-pub-key = ""
+
+    system-contract-pub-key = ""
```

Comment:
@zsluedem Please create an issue to add description for this configuration parameter.